files/default/github_pr_comment_trigger.rb

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env ruby
+
+# Takes a JSON payload from a Github PR issue webhook from stdin, reads the
+# comment, and if it matches `!bump (major|minor|patch)?` (defaults to
+# `minor`), merges the PR, bumps the Chef cookbook's metadata.rb, creates a
+# version tag, and pushes up.
+
+require 'git'
+require 'json'
+require 'octokit'
+require 'pp'
+
+GIT_BRANCH = 'master'
+METADATA_FILE = 'metadata.rb'
+COMMAND = '!bump'
+LEVELS = {
+  'major' => 0,
+  'minor' => 1,
+  'patch' => 2
+}
+
+d = JSON.load(STDIN.read)
+
+# Check if we got a valid request and get the bump level
+comment = d.fetch('comment', {}).fetch('body', '')
+match = comment.match(/^#{COMMAND}( (#{LEVELS.keys.join('|')}))?$/, 2)
```

Comment:
Is there a reason why you create one match group with a space and then the other match group?